### PR TITLE
Add guide for uploading and distributing vagrant boxes

### PR DESCRIPTION
**Pull request by KFishner**

### CODE REVIEW

**Comment by pearkes** [comment]
I think you need to elaborate on the "release" step of the version.

File: _nav.html.erb [hunk 1]
```diff
@@ -37,6 +37,7 @@
     <li class="<%= "active" if help_active?("guides") %>">
         <h3 id="guides" class="h4"><%= help_link "Guides: How to use Atlas with your existing services", "guides/continuous-integration" %></h3>
         <ul>
+            <li><%= help_link "How to use Atlas to store and distribute Vagrant boxes", "guides/vagrant-box-create-and-distribute" %></li>
```
Comment (sethvargo) [inline _nav.html.erb]: "How to use to" feels really weird to be. Can we change these verbs to "Using" - "Using Atlas to store and distribute Vagrant boxes"

File: guides/vagrant-box-create-and-distribute.html.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Once you've created and released a box, you can release new versions of
+the box by clicking "Create New Version" under the versions dropdown of
+a box page. For more information on the release lifecycle of boxes, see
+the [help page dedicated to it](/help/boxes/lifecycle).
+
+To distribute the box to your team, update your Vagrantfile to reference the
+box on Atlas.
+
+	Vagrant.configure(2) do |config|
```
Comment (sethvargo) [inline guides/vagrant-box-create-and-distribute.html.md]: We should use the tickticktick (`````) here with "ruby" so we get the syntax highlighting

Comment (KFishner) [inline guides/vagrant-box-create-and-distribute.html.md]: this didn't work :(

Comment (sethvargo) [inline guides/vagrant-box-create-and-distribute.html.md]: :panda_face: ok

**Pull request merged**